code/game/objects/items/food/snacks.dm

```diff
@@ -206,14 +207,20 @@
 	food_reagents = list(/datum/reagent/consumable/nutriment = 2, /datum/reagent/consumable/bbqsauce = 1)
 	tastes = list("peanuts" = 3, "bbq sauce" = 1, "arguments" = 1)
 
+/obj/item/food/peanuts/ban_appeal
+	name = "\improper Gallery's peanuts Ban Appel mix"
+	desc = "An ill-fated attempt at trail mix, banned in 6 sectors. Yearly lobbying to overturn is denied not because the apples are toxic, but because they keep evading the ban."
+	food_reagents = list(/datum/reagent/consumable/nutriment = 2, /datum/reagent/toxin/cyanide = 1) //uses dried poison apples
+	tastes = list("peanuts" = 3, "apples" = 1, "regret" = 1)
+
 /obj/item/food/peanuts/random
 	name = "\improper Gallery's every-flavour peanuts"
 	desc = "What flavour will you get?"
 	icon_state = "peanuts"
 
 /obj/item/food/peanuts/random/Initialize()
 	// Generate a sample p
-	var/peanut_type = pick(subtypesof(/obj/item/food/peanuts) - /obj/item/food/peanuts/random)
+	var/peanut_type = pick(subtypesof(/obj/item/food/peanuts) - /obj/item/food/peanuts/random - /obj/item/food/peanuts/ban_appeal)
```

Comment:
Why can't this be in the random peanut list?
If it's dangerous not to, then a flag would be better than removing it from the list directly.
You can populate the list like this, for example:
```dm
GLOBAL_LIST(safe_peanut_types) = populate_safe_peanut_types()

/proc/populate_safe_peanut_types()
	. = list()
	for(var/obj/item/food/peanuts/peanut_type as anything in subtypesof(/obj/item/food/peanuts))
		if(!initial(peanut_type.safe_for_consumption))
			continue
		. += peanut_type
```

This implies you add a `safe_for_consumption` variable that is `TRUE` by default and `FALSE` for the random peanuts and these you're adding.

Then you can do `pick(GLOB.safe_peanut_types)` instead.

Reply:
Why can't I? because I don't know how to code it and that was conveniently already there to add to